_includes/expand-parameter.md

```diff
@@ -0,0 +1,26 @@
+### Expansion
```

Comment:
As we discussed, should we have headers in includes?
Static headers will be an issue if we implement the includes on different header levels.

Reply:
Perhaps we can send in header start level as a parameter and have the include file use [liquid code](https://jekyllrb.com/docs/liquid/) to output the headers dynamically?

Reply:
Perhaps, it's something we can look into